pages/careers/index.tsx

```diff
@@ -15,13 +16,40 @@ import Prose from "../../components/Prose/Prose";
 import clsx from "clsx";
 import { getHeadPageTitle } from "../../utils/meta";
 import { PageSettings } from "../../types/next";
+import {
+  getNewsEntries,
+  getCustomPageAndMicrocopy,
+} from "../../contentful/client";
+import {
+  TypeNews,
+  TypeMicrocopy,
+  TypeCustomPage,
+} from "../../generated/contentful";
+import Microcopy from "../../components/Microcopy/Microcopy";
 
 export const pageSettings: PageSettings = {
   path: "/careers",
   linkContent: <>Careers</>,
 };
 
-const Index = (): ReactElement => {
+export const getStaticProps: GetStaticProps<{
+  microcopy: TypeMicrocopy[];
+  customPage?: TypeCustomPage;
+}> = async () => {
+  const content = await getCustomPageAndMicrocopy("acalTvHwciWsZovjVPPEL");
```

Comment:
i'd prefer if the ids came from an object in a file in the `contentful` dir. then they can be given meaningful names